test/failover/cluster_changes.result

```diff
@@ -54,7 +54,7 @@ test_run:switch('router_1')
 -- First test case: reverse weights, when only replica exists,
 -- and no replica candidate.
 --
-vshard.router.cfg(cfg)
+util.router_cfg(cfg)
```

Comment:
Lets make it more explicit that this function also calls `box.cfg` by naming it `util.box_router_cfg`.